stateproof/builder.go

```diff
@@ -155,6 +180,13 @@ func (spw *Worker) createBuilder(rnd basics.Round) (builder, error) {
 		return builder{}, err
 	}
 
+	err = spw.db.Atomic(func(_ context.Context, tx *sql.Tx) error {
+		return persistBuilder(tx, rnd, &res)
+	})
+	if err != nil {
```

Comment:
If this code fails during handleSig the handler will not return an error, which is not the behavior we would like to have. 

I suggest that `createBuilder` will not log and just return the error. `OnPrepareVoterCommit` should log instead and not report the error.

Reply:
Decided to keep this behavior.